main.go

```diff
@@ -185,8 +224,9 @@ func handleMessageEvent(rtm *slack.RTM, event *slack.MessageEvent) error {
 }
 
 func checkMessageEvent(event *slack.MessageEvent) error {
+	// skip topic messages
 	if event.Topic != "" {
-		return fmt.Errorf("inocming message about topic change")
+		return fmt.Errorf("incoming message about topic change")
```

Comment:
`the incoming message about topic change`
or
`an incoming message about topic change`